packages/ui/src/common/components/BlockTime/BlockHistoryLine.tsx

```diff
@@ -1,21 +1,22 @@
 import React from 'react'
 import styled from 'styled-components'
 
-import { Tooltip, TooltipDefault } from '@/common/components/Tooltip'
+import { BlockTime } from '@/common/components/BlockTime'
 import { TextInlineSmall } from '@/common/components/typography'
 import { Fonts } from '@/common/constants'
-import { formatDateString } from '@/common/model/formatters'
 
-import { BlockInfo, BlockInfoProp } from './BlockInfo'
+// import { formatDateString } from '@/common/model/formatters'
+import { BlockInfoProp } from './BlockInfo'
 
-export const BlockHistoryLine = ({ block }: BlockInfoProp) => (
-  <BlockHistoryLineContainer>
-    <TextInlineSmall lighter>{formatDateString(block.timestamp)}</TextInlineSmall>
-    <Tooltip popupContent={<BlockInfo block={block} />}>
-      <TooltipDefault />
-    </Tooltip>
-  </BlockHistoryLineContainer>
-)
+export const BlockHistoryLine = ({ block }: BlockInfoProp) => {
```

Comment:
@chrlschwb the ui looks good but is this component still necessary ? What happens if you just replace `details` in `@/proposals/components/ProposalHistory` by: `<BlockTime block={inBlock} layout="reverse-start" lessInfo />` ?